config/routes.rb

```diff
@@ -300,4 +292,6 @@
 
   # See how all your routes lay out with "rake routes"
 
+  # Legacy routes maintained to avoid routes breaking
+  get "/forum/*anything" => "pages#forum"
```

Comment:
Nit: This doesn't match /forum or /forum/, no big deal.

Reply:
hmm thats annoying tho - i'll fix it. the entire point is to avoid dead SEO links or bookmarks, as unlikely as they may be.

Reply:
thanks for catching this - have fixed and added a spec